src/inc/msquic.h

```diff
@@ -197,6 +202,27 @@ typedef struct QUIC_REGISTRATION_CONFIG { // All fields may be NULL/zero.
     QUIC_EXECUTION_PROFILE ExecutionProfile;
 } QUIC_REGISTRATION_CONFIG;
 
+typedef
+_IRQL_requires_max_(PASSIVE_LEVEL)
+_Function_class_(QUIC_CREDENTIAL_LOAD_COMPLETE)
+void
+(QUIC_API QUIC_CREDENTIAL_LOAD_COMPLETE)(
+    _In_ HQUIC Configuration,
+    _In_opt_ void* Context,
+    _In_ QUIC_STATUS Status
+    );
+
+typedef QUIC_CREDENTIAL_LOAD_COMPLETE *QUIC_CREDENTIAL_LOAD_COMPLETE_HANDLER;
+
+typedef struct QUIC_CREDENTIAL_CONFIG {
+    QUIC_CREDENTIAL_TYPE Type;
+    QUIC_CREDENTIAL_FLAGS Flags;
+    void* Creds;
```

Comment:
Possibly replace with a pointer to a union (`QUIC_CERTIFICATE`?) of all other structs below?